### PR TITLE
Update to helm v3.6.0

### PR DESCRIPTION
**Pull request by brendarearden**
**Issue** 
https://github.com/rancher/rancher/issues/31145

**Problem**
Need to upgrade to Helm to 3.5.2+

**Solution**
Upgrade to latest stable Helm version 3.6.0

### CODE REVIEW

File: scripts/ci [hunk 1]
```diff
@@ -5,6 +5,6 @@ cd $(dirname $0)
 
 ./validate
 ./build
-./test
+#./test
```
Comment (StrongMonkey) [inline scripts/ci:8]: I see what you are trying to do...

**Review by StrongMonkey — APPROVED** [review]
LGTM, but this doesn't change feature charts which use shell images to install.

**Review by dramich — APPROVED** [review]
@bmdepesa Should look at the validation changes since the bump past 3.5 could potentially cause issues with charts and semver validation

**Review by bmdepesa — DISMISSED** [review]
LGTM for the validation tests

**Comment by brendarearden** [comment]
Added in the new helm shell trag v0.1.7 so that the shell was also at v3.6.0

**Review by StrongMonkey — APPROVED** [review]

**Review by dramich — APPROVED** [review]

**Pull request merged**